pkg/csoclients/csoclients.go

```diff
@@ -286,3 +307,33 @@ func StartMgmtInformers(clients *Clients, stopCh <-chan struct{}) {
 		informer.Start(stopCh)
 	}
 }
+
+func extractOperatorSpec(obj *unstructured.Unstructured, fieldManager string) (*applyoperatorv1.OperatorSpecApplyConfiguration, error) {
+	castObj := &operatorv1.Storage{}
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(obj.Object, castObj); err != nil {
+		return nil, fmt.Errorf("unable to convert to ClusterCSIDriver: %w", err)
```

Comment:
Please update the log message here and in `extractOperatorStatus` to replace `ClusterCSIDriver` with `Storage`.